src/index.md

```diff
@@ -1,9 +1,17 @@
 # The Brain Imaging Data Structure
 
-This resource defines the Brain Imaging Data Structure (BIDS) specification, including the core specification as well as many modality-specific extensions.
+The Brain Imaging Data Structure (BIDS) is a simple and intuitive way to
+organize and describe data.
 
-To get started, [check out the introduction](01-introduction.md). If you'd like
-more information on how to adapt your own datasets to match the BIDS
-specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+This document defines the BIDS specification, which provides all of the details
+to implement the standard. It includes the core specification as well as many extensions
```

Comment:
```suggestion
This document defines the BIDS specification, which provides many details
to help implement the standard. It includes the core specification as well as many extensions
```

I will split hair on this one, but unfortunately the spec does not provide "all" the details. I mean I wished it did otherwise we would not have to regularly open that many PR to add more details regarding some blind spots we had not thought of.

Similarly the spec still leave a lot of flexibility on how to implement it on a dataset.

So I am just "hedging" that sentence a bit.

Reply:
Fair enough :-)